src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -208,6 +209,29 @@ internal CookieContainer CookieContainer
             }
         }
 
+        internal int MaxAutomaticRedirections
+        {
+            get
+            {
+                return _maxAutomaticRedirections;
+            }
+
```

Comment:
removed extra blank line in set block. Maintaining one line gap between get & set.